lms/static/js/staff_debug_actions.js

```diff
@@ -17,95 +19,21 @@ var StaffDebug = (function(){
         uname =  $('#sd_fu_' + locname).attr('placeholder');
     }
     return uname;
-  }
-
-  do_idash_action = function(action){
```

Comment:
I'm having a hard time picking apart what's going on in this file, and how your changes relate to that. Why have you removed all these methods?